Dockerfile

```diff
@@ -1,18 +1,13 @@
-FROM node:19.5
+FROM node:19.5.0-alpine
```

Comment:
use the LTS version of node or the same version you have in your docker-compose file. @musllim